upload.js

```diff
@@ -15,12 +18,20 @@ function upload (opts, cb) {
     })
   }
 
+  var semver = SemVer(pkg.version)
+  var prerelease = opts.prerelease
+  if (prerelease && prerelease !== true) semver.prerelease.push(prerelease)
```

Comment:
When false (the default) or falsy value, the release is only marked as
prerelease if the package.json version field has it defined; if true or
truthty vamue, it force the release to be a prerelease; and if a string or
a number, it also append its value to the chain of the prerelease field.
Maybe should I add it in the docs or in a comment? Should I also add a
check to allow zero and disallow empty string?
El 13/6/2016 8:36 AM, "Julian Gruber" <notifications@github.com> escribió:

> In upload.js
> <https://github.com/mafintosh/prebuild/pull/127#discussion_r66744915>:
>
> > @@ -15,12 +18,20 @@ function upload (opts, cb) {
> >      })
> >    }
> >
> > +  var semver = SemVer(pkg.version)
> > +  var prerelease = opts.prerelease
> > +  if (prerelease && prerelease !== true) semver.prerelease.push(prerelease)
>
> why should it be not true?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mafintosh/prebuild/pull/127/files/beef82503dbb1296d7606e551546b6a2f26333c4#r66744915>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgfvqCNC70x7jwzdQyntx17XC0vG9m7ks5qLPqIgaJpZM4Iz52O>
> .
>